package.json

```diff
@@ -95,30 +95,29 @@
     "write-file-atomic": "^3.0.1"
   },
   "devDependencies": {
-    "@types/cosmiconfig": "^5.0.3",
     "@types/debug": "^4.1.5",
     "@types/global-modules": "^2.0.0",
     "@types/globjoin": "^0.1.0",
-    "@types/lodash": "^4.14.146",
+    "@types/lodash": "^4.14.149",
     "@types/meow": "^5.0.0",
-    "@types/micromatch": "^3.1.0",
+    "@types/micromatch": "^3.1.1",
     "benchmark": "^2.1.4",
     "common-tags": "^1.8.0",
     "cp-file": "^7.0.0",
     "del": "^5.1.0",
-    "eslint": "~6.6.0",
-    "eslint-config-stylelint": "~11.1.0",
+    "eslint": "^6.7.1",
+    "eslint-config-stylelint": "^11.1.0",
```

Comment:
Using `^` caret here has caused us problems in the past, I think continuing to use `~` tillde here would be preffered please

Reply:
@ntwb with a lock file this is moot. The behavior does not change, unless there's a security fix